tests/test_smoke_eval.py

```diff
@@ -66,51 +59,42 @@ def eval_run(elicit: Elicit, tfr_datasets: list[str] = None) -> int:
     # record elicit modification time as reference.
     start_time_sec = (tmp_path / "eval.csv").stat().st_mtime
 
-    if tfr_datasets:
+    if transfer_datasets:
         # update datasets to a different dataset
-        extract.prompts.datasets = tfr_datasets
+        extract.prompts.datasets = transfer_datasets
 
     eval = Eval(data=extract, source=tmp_path)
     eval.execute()
     return start_time_sec
 
 
-def eval_assert_files_created(elicit: Elicit, start_time_sec=0):
+def eval_assert_files_created(elicit: Elicit, transfer_datasets: Sequence[str] = []):
     tmp_path = elicit.out_dir
```

Comment:
eval's execute function, on the latest branch, will only run for the specified transfer datasets so theres no self eval now